layouts/partials/footer.html

```diff
@@ -4,5 +4,8 @@
       <p>{{.}}</p>
     {{ end }}
     {{ if not .Site.Params.hideCopyright }} © {{ .Site.LastChange.Format "2006" }} {{ end }} {{ if not .Site.Params.hideCredits}} {{ if not .Site.Params.hideCopyright }} · {{ end }} Powered by <a href="https://gohugo.io/">Hugo</a> & <a href="https://github.com/luizdepra/hugo-coder/">Coder</a>. {{ end }}
```

Comment:
You need to remove this line. Everything is duplicated.
Look at the PR preview -> https://deploy-preview-78--hugo-coder.netlify.com/

Reply:
Not sure how I managed that, but I am working on it.

Reply:
Sorry for all the hassles, and Thanks for being patient with me.

Reply:
Still not sure if I got it. If it's still messed up, I will take a look this coming weekend when I have more time.